runtime/compiler/net/CommunicationStream.hpp

```diff
@@ -45,11 +45,14 @@ class CommunicationStream
 
    static uint32_t _msgTypeCount[MessageType::MessageType_MAXTYPE];
    static uint64_t _totalMsgSize;
+   static uint32_t _lastReadError;
```

Comment:
Do we have any synchronization concerns because these are static and can me modified concurrently by different threads?

Reply:
I have not yet thought about this aspect. Need to discuss this further.